### ISSUE TITLE
PHP Error running setup.php

### ISSUE THREAD

**Issue by A1Gaming** [postfixadmin/postfixadmin#698]
Hello, 

Upon setting up Postfix Admin, and following the INSTALL.TXT file's instructions, I am encountering an error running the setup.php file. This error only appeared because I set display_errors = On in the PHP.ini config file. Otherwise, it would just display a blank page.

---START ERROR MESSAGE---

Warning: require_once(/srv/postfix-admin/vendor/autoload.php): Failed to open stream: No such file or directory in /srv/postfix-admin/common.php on line 3

Fatal error: Uncaught Error: Failed opening required '/srv/postfix-admin/vendor/autoload.php' (include_path='.:/usr/share/pear:/usr/share/php:/usr/share/pear:/usr/share/php') in /srv/postfix-admin/common.php:3 Stack trace: #0 /srv/postfix-admin/public/common.php(3): require_once() #1 /srv/postfix-admin/public/login.php(30): require_once('...') #2 {main} thrown in /srv/postfix-admin/common.php on line 3

---END ERROR MESSAGE---

I made sure SELinux is not blocking setup.php file. The /srv/postfix-admin permissions were not modified out of the box, and its owner is apache:apache

The machine is running Rocky Linux 8.7 and PHP version 8.1.13. My first thoughts are is Postfix Admin compatible with this version of php?

**Comment by DavidGoodwin** [comment]
Hello @A1Gaming 

I think you're seeing that because you're using the 'master' branch of PostfixAdmin (that's fine, but i don't think you'd have seen it if you were using a stable v3.3 release).

The fix should be to run the 'install.sh' shell script.

No doubt INSTALL.TXT needs updating though ....

**Comment by A1Gaming** [comment]
Thanks for the quick reply. I should of mentioned how I got the program...I used this command:
'sudo wget -O postfixadmin.tgz https://github.com/postfixadmin/postfixadmin/tarball/master'

Which is found at this webpage: 'https://postfixadmin.github.io/postfixadmin/'

The filename is 'postfixadmin-postfixadmin-postfixadmin-3.3.8-486-gfcf14f0.tar.gz', so the version I'm using is 3.3.8-486.

So, yes, it is the "master" branch, my apologies for that. I should of paid better attention. You did explain the proper steps in your INSTALL.TXT file.

I went ahead and ran the install.sh shell script as root, and now everything works perfect. Setup updated the existing database from version 1844 to 1848 without a hitch. The environment variables are all green checks. Everything is working perfectly, as I can tell.

Again, I guess I shouldn't of starting working on this project late at night! Thanks for your time though!
